src/index.html

```diff
@@ -5,8 +5,391 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="container">
+      <header class="header">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+  
+        <nav class="navigation">
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link is-active">Apple</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Samsung</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a href="#home" class="navigation__link">Smartphones</a>
+            </li>
+          </ul>
+          <ul>
+            <li class="navigation__li">
+              <a
+                href="#home"
+                class="navigation__link"
+                data-qa="nav-hover"
+              >Laptops & computers</a>
```

Comment:
Improve your code style everywhere
```suggestion
              <a
                href="#home"
                class="navigation__link"
                data-qa="nav-hover"
              >
                Laptops & computers
              </a>
```